package.json

```diff
@@ -27,10 +27,16 @@
     },
     "keywords": [],
     "dependencies": {
-        "http-status-codes": "^2.1.4",
-        "winston": "^3.3.3"
+        "axios": "^0.24.0",
+        "fastify": "^3.22.1",
+        "http-status-codes": "^2.1.4"
```

Comment:
you don't need this
see https://nodejs.org/api/http.html#httpstatus_codes

Reply:
Thank you. I know about this feature. It's more convenient and readable, from my point of view, to use syntax like:

`StatusCodes.BAD_GATEWAY`

instead of just

`502`

for example. Furthermore, imo, rule about "magic numbers" relates also to bad known/less used standardized codes, because for those who see them first time they cannot be distinguished from custom.